docs/SviatoslavTaranenko/Games.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
```

Comment:
The best practice in file naming is lowercase without spaces. I don't like to insist you rename all your files, it could be a tricky task (you can try if you like). Just as an advice for you.
Here is a long description from ChatGPT why it is important:

Using spaces and capitalized file names can potentially lead to issues on different operating systems due to variations in how file systems handle naming conventions. Here are some potential problems associated with spaces and capitalized file names on different operating systems:

Windows:
Case Insensitivity:

Windows file systems are case-insensitive by default. This means that files named, for example, "File.txt" and "file.txt" will be treated as the same file. This can lead to confusion and unexpected behavior.
Command Line and Scripts:

In command-line environments, scripts, and batch files, having spaces in file names may require additional quoting or escaping to ensure proper parsing. It can complicate automation scripts.
Unix-based Systems (Linux, macOS):
Case Sensitivity:

Unix-based file systems are case-sensitive. "File.txt" and "file.txt" are considered as different files. If a project is developed on a case-insensitive system (e.g., Windows) and then moved to a case-sensitive system, issues may arise.
URLs and Web Development:

In web development, URLs are case-sensitive. If a file is referenced with different capitalization in code than what's on the file system, it may lead to broken links and issues in web applications.
Shell Commands and Scripts:

Shell commands and scripts may need adjustments when working with files containing spaces or capitalized letters. Quoting or escaping may be necessary to handle such file names.
Common Issues on Both Systems:
Compatibility with Tools and Libraries:

Some development tools, libraries, and frameworks may have limitations or conventions regarding file names. Consistent lowercase letters without spaces are often recommended for compatibility.
Version Control Systems:

Version control systems, such as Git, may handle file names differently based on the underlying file system. Renaming files with different capitalization may cause issues, and repositories may not work consistently across platforms.
URLs in Documentation:

If file names are referenced in documentation, having spaces or inconsistent capitalization can lead to broken links and confusion.
Recommendations:
Consistency is Key:

It's crucial to maintain consistency within a project. Choose a naming convention and stick to it throughout the entire codebase.
Avoid Spaces and Capitalization:

To ensure broad compatibility, it's generally advisable to avoid spaces and capitalized letters in file names. Instead, use underscores, hyphens, or camelCase for word separation.
Follow Platform-Specific Guidelines:

Adhere to platform-specific guidelines and best practices. Check the recommendations for the operating systems your project is intended to run on.
By following consistent naming conventions and being aware of platform-specific considerations, you can minimize potential issues related to spaces and capitalized file names across different operating systems.